### PR TITLE
pre chat survey pane narrator issue

### PR DESCRIPTION
**Pull request by v-hghildiyal**
[Bug 3047419](https://dev.azure.com/dynamicscrm/OneCRM/_workitems/edit/3047419): A11y_Live Chat Widget - Version 2_Live Chat_conversation with pre-chat_Screen reader: NVDA/Narrator is announcing the same information, "Fields marked with * are mandatory", twice

Reason :- Message was appended again in button parent div aria-label , so it was reading again. As in attached screenshot.

Fix:- added condition in PreChatSurveyPaneStateful where aria-label is set to field so that it exclude div with class "ac-actionSet".
Now narrator reads only once.

image

### CODE REVIEW

**Review by charliewang95 — APPROVED** [review]

File: chat-widget/src/components/prechatsurveypanestateful/PreChatSurveyPaneStateful.tsx [hunk 1]
```diff
@@ -117,7 +117,10 @@ export const PreChatSurveyPaneStateful = (props: IPreChatSurveyPaneStatefulParam
                 }
                 if (current && current.tagName.toLowerCase() == HtmlAttributeNames.div && current.childElementCount > 0) {
                     const input = current.children[0].children;
-                    if (input && input.length > 0 && input[0].className != HtmlAttributeNames.adaptiveCardToggleInputClassName) {
+                    if (input
+                        && input.length > 0
```
Comment (charliewang95) [inline chat-widget/src/components/prechatsurveypanestateful/PreChatSurveyPaneStateful.tsx:121]: Can simplify to `input?.length`?

**Review by charliewang95 — COMMENTED** [review]
Please modify CHANGE_LOG.md

**Review by elopezanaya — CHANGES_REQUESTED** [review]
Fix should work for NVDA and Narrator

File: chat-widget/src/components/prechatsurveypanestateful/PreChatSurveyPaneStateful.tsx (hunk 1 above)
Comment (v-hghildiyal) [reply chat-widget/src/components/prechatsurveypanestateful/PreChatSurveyPaneStateful.tsx]: Applied as mentioned.

**Pull request merged**